### ISSUE TITLE
[arm64] Current status on rpi4

### ISSUE THREAD

**Issue by swift-kim** [flutter-tizen/flutter-tizen#83]
- [ ] The app takes up only 2/3 of the display.

  ![20150322-125958](https://user-images.githubusercontent.com/40190588/114368308-45119500-9bb8-11eb-9a47-f25281a76b76.jpg)

- [x] Building native tpk fails with the "file format not recognized" error.

  ```sh
  $ flutter-tizen run
  ...
    Building target: Debug/runner
    Invoking: C/C++ Linker
  "/home/swift/tizen-studio/tools/smart-build-interface/../llvm-10/bin/clang++" -o ./Debug/runner @./Debug/target_objs.file   -Llib
  -L"/home/swift/Git/flutter/dummy/arm64_native/tizen/lib" -Wl,--no-undefined  -Wl,-E -target arm-tizen-linux-gnueabi -gcc-toolchain
  "/home/swift/tizen-studio/tools/smart-build-interface/../arm-linux-gnueabi-gcc-9.2/" -ccc-gcc-name arm-linux-gnueabi-g++ -march=armv7-a -mfloat-abi=softfp
  -mfpu=vfpv3-d16 -mtune=cortex-a8 -Wl,-z,relro,--as-needed  -Xlinker --as-needed -pie -lpthread
  --sysroot="/home/swift/tizen-studio/tools/smart-build-interface/../../platforms/tizen-5.5/wearable/rootstraps/wearable-5.5-device.core" -Xlinker
  --version-script="./.exportMap" -L/home/swift/tizen-studio/platforms/tizen-5.5/wearable/rootstraps/wearable-5.5-device.core/usr/lib @./Debug/platform_libs.file
  -Xlinker -rpath='$ORIGIN/../lib' -Werror-implicit-function-declaration -lflutter_tizen -L/home/swift/Git/flutter/dummy/arm64_native/tizen/lib
  -I/home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/client_wrapper/include
  -I/home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/public -I/home/swift/Git/flutter/flutter-tizen/embedding/cpp/include -DCOMMON_PROFILE
  -Wl,-unresolved-symbols=ignore-in-shared-libs
  lib/libflutter_tizen.so: file not recognized: file format not recognized
  clang++: error: linker command failed with exit code 1 (use -v to see invocation)
  make[1]: *** [/home/swift/tizen-studio/tools/ide/resources/native/Build/makefile.mk:135: Debug/runner] Error 1
  make[1]: Leaving directory '/home/swift/Git/flutter/dummy/arm64_native/tizen'
  make: *** [/home/swift/tizen-studio/tools/ide/resources/native/Build/makefile:27: app_build] Error 2
  Total time: 00:00:04.719
  ``

- [x] Set up continuous integration for arm64 engine release.

**Comment by swift-kim** [comment]
The second issue could have been resolved by adding the Tizen 6.0 aarch64 rootstrap (#84), but still there's another issue:

```
...
  Building file: /home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/client_wrapper/core_implementations.cc
  Output Path ./Release
  Invoking: C++ Compiler
  Building directory: Release//home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/client_wrapper
"/home/swift/tizen-studio/tools/smart-build-interface/../llvm-10/bin/clang++" -c
"/home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/client_wrapper/core_implementations.cc" -o
"Release//home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/client_wrapper/core_implementations.o" -DTIZEN_DEPRECATION
-DDEPRECATION_WARNING  -DTIZEN_DEPRECATION -DDEPRECATION_WARNING  -Iinc -Iflutter -Isrc  -I"pch" -O3 -Wall -c -fmessage-length=0 -std=c++11 -target
aarch64-tizen-linux-gnu -gcc-toolchain "/home/swift/tizen-studio/tools/smart-build-interface/../aarch64-linux-gnu-gcc-9.2/" -ccc-gcc-name aarch64-linux-gnu-g++
-march=armv8-a+fp+simd+crc+crypto -Wno-gnu -fstack-protector-strong -Wno-unused-command-line-argument -fdebug-default-version=3 -std=c++17 -fPIE
--sysroot="/home/swift/tizen-studio/tools/smart-build-interface/../../platforms/tizen-6.0/wearable/rootstraps/wearable-6.0-device.core"
-Werror-implicit-function-declaration  -lflutter_tizen -L/home/swift/Git/flutter/dummy/arm64_native/tizen/lib
-I/home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/client_wrapper/include
-I/home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/public -I/home/swift/Git/flutter/flutter-tizen/embedding/cpp/include -DCOMMON_PROFILE
-Wl,-unresolved-symbols=ignore-in-shared-libs -Wp,@./Release/objs/platform_incs_file.inc
In file included from /home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/client_wrapper/core_implementations.cc:21:
In file included from /home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/client_wrapper/include/flutter/engine_method_result.h:8:
/home/swift/tizen-studio/tools/smart-build-interface/../aarch64-linux-gnu-gcc-9.2/lib/gcc/aarch64-tizen-linux-gnu/9.2.0/../../../../aarch64-tizen-linux-gnu/inc
lude/c++/9.2.0/memory:121:25: error: cast from pointer to smaller type 'uintptr_t' (aka 'unsigned int') loses information
  const auto __intptr = reinterpret_cast<uintptr_t>(__ptr);
                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[1]: *** [/home/swift/tizen-studio/tools/ide/resources/native/Build/makefile.mk:97:
Release//home/swift/Git/flutter/flutter-tizen/bin/cache/artifacts/engine/common/client_wrapper/core_implementations.o] Error 1
make[1]: Leaving directory '/home/swift/Git/flutter/dummy/arm64_native/tizen'
make: *** [/home/swift/tizen-studio/tools/ide/resources/native/Build/makefile:27: app_build] Error 2
Total time: 00:00:01.746
```

Maybe the Tizen CLI builder doesn't fully support the aarch64 architecture yet.

**Comment by swift-kim** [comment]
Regarding the first problem, we can obtain the real screen size using `ecore_wl2_display_screen_size_get` (ecore_wl2) and `ecore_evas_screen_geometry_get` (Evas GL). The embedder API and implementation will be changed soon based on this.
